packages/worker/src/workflow/bundler.ts

```diff
@@ -315,6 +320,12 @@ export interface BundleOptions {
    * > NOTE: This is an advanced option that should be used with care.
    */
   ignoreModules?: string[];
+
+  /**
+   * Before Workflow code is bundled with Webpack, `webpackConfigHook` is called with the Webpack
+   * {@link https://webpack.js.org/configuration/ | configuration} object so you can modify it.
+   */
+  webpackConfigHook?: (config: webpack.Configuration) => void;
```

Comment:
As it is now, `webpackConfigHook` is defined to return `void`, but everywhere, the code expects it to actually return the configuration.

Reply:
Fixed